src/state/store.js

```diff
@@ -1,11 +1,17 @@
 import thunk from "redux-thunk";
 import {combineReducers, createStore, applyMiddleware, compose} from "redux";
 import authReducer from "./reducers/auth";
-import deckReducer from "./reducers/decks";
+import {
+  publicDecksReducer,
+  personalDecksReducer,
+  currentDeckReducer,
+} from "./reducers/decks";
 
 const mainReducer = combineReducers({
   authState: authReducer,
-  deckState: deckReducer,
+  publicDeckState: publicDecksReducer,
+  personalDeckState: personalDecksReducer,
+  currentDeckState: currentDeckReducer,
 });
```

Comment:
This is not how the state is handled on develop, look at how it's done there.
deckState should contain all the 3 other slices